lib/opentsdb-consumer/metric.rb

```diff
@@ -12,7 +14,8 @@ def initialize(name: nil, aggregator: 'sum', rate: false, downsample: '10m-avg',
 
     def to_s
       query = tags.any? ? "{#{tags_to_query}}" : ''
-      [aggregator, downsample, rate_to_query, name].compact.join(':') + query
+      query += ":#{rate.to_s}" if rate
```

Comment:
`to_s` ist zwar schön explizit, in String Interpolations aber überflüssig. Das macht Ruby von alleine.